helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record model="ir.module.category" id="module_helpdesk_category">
+            <field name="name">Helpdesk</field>
+            <field name="description">Helps you handle your helpdesk security.</field>
+            <field name="sequence">9</field>
+        </record>
+    </data>
+    <data noupdate="1">
+        <!--Email template -->
+        <record id="assignment_email_template" model="mail.template">
+            <field name="name">Ticket Assignment</field>
+            <field name="model_id" ref="model_helpdesk_ticket" />
+            <field name="email_from">${object.company_id.partner_id.email}</field>
+            <field
```

Comment:
Add line email_cc
<field name="email_cc">${not object.partner_id and object.partner_email or ''|safe},</field>
See https://github.com/OCA/helpdesk/pull/98